config/prometheus/prometheus-roles.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
Sorry if I am getting too picky, but we are only monitoring resources in `argo` namespace right? So should we use `Role` instead of `ClusterRole`?

Reply:
No worries. I think we're using the CusterRole so that Prometheus can monitor across namespaces.